README.md

```diff
@@ -1,9 +1,9 @@
 <!-- Documentation start -->
 <!-- Logo start -->
-<img align="right" width="100" src="https://gcdnb.pbrd.co/images/nlDr0mgn0Nkp.png"></a>
+<img align="right" width="120" height="100" src="https://gcdnb.pbrd.co/images/nlDr0mgn0Nkp.png"></a>
```

Comment:
- width should be 100
- delete height

Reply:
Insert this code instead of the code in line 3:
```HTML
<img align="right" width="100" src="https://gcdnb.pbrd.co/images/nlDr0mgn0Nkp.png"></a>
```

Reply:
right please do it 
cc @KartikKar19